### ISSUE TITLE
How to divide the data set？

### ISSUE THREAD

**Issue by Haitons** [Websail-NU/torch-defseq#5]
Hello Thanapon，how to divide dictionary corpus into training set,development set ,test set and ensure that Word, Entries, tokens, Avg length are suitable?I found this a bit difficult

**Comment by northanapon** [comment]
I divided the datasets randomly, but I kept the same words in the same corpus. For example, all dictionary entries of a word "tie" will be in a set. As for the length, I think we just need a large number of tokens for training.

**Comment by Haitons** [comment]
Ok.Thank you.
